admin/javascript/src/sections/campaign-admin/styles.scss

```diff
@@ -1,3 +1,210 @@
 .CampaignAdmin {
-    overflow-y: auto;
-}
\ No newline at end of file
+	overflow-y: auto;
+	display: block;
+
+	// Contains campaign form and preview layout
+	.cms-middle {
+		padding-left: 0;
+		height: 100%;
+		position: relative;
+		transition: padding .2s;
+
+		&.with-preview {
+			@media (min-width: 992px) { /* lg */
+				padding-left: 316px;
+				.cms-campaigns {
+					width: 316px;
+				}
+			}
+			@media (min-width: 1200px) { /* xl */
+				padding-left: 448px;
+				.cms-campaigns {
+					width: 448px;
+				}
+			}
+		}
+	}
+
+	/* CAMPAIGNS */
+	.cms-campaigns {
+		width: 100%;
+		position: absolute;
+		left: 0;
+		top: 0;
+		height: 100%;
+		overflow: hidden;
+		background-color: #f6f7f8;
+		z-index: 2;
+		transition: width .2s;
+		padding-bottom: 53px;
+
+		.campaign-items {
```

Comment:
Most of this CSS here should move into the `<Accordion>` component, which @clarkepaul is working on right now.